examples/cloud-composer-datafusion-ingestion-templates/composer/dag-dependencies/read_params.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" This module reads the parameters from all three parameter files-
+environment parameter, dag paramter and task parameter
+"""
+import json
+import logging
+import os
+from airflow.models import Variable
+from airflow import AirflowException
+
+class ParamReader:
+    """ParamReader class reads parameters from the 3 parameter files
+    function read_parameters:
+        Reads parameters from param files
+        and assigns the values to the class variables.
+        This is the main function
+        to be called whenever parameters need to be read.
+    """
+    env_param_file=""
```

Comment:
why are you setting these as class attributes? wouldn't these be instance attributes?